### PR TITLE
Average Minutemen Facility Redux

### PR DESCRIPTION
**Pull request by Latentish**
## About The Pull Request

cmmbase
Adds the Colonial Minutemen Xenohazard Research Compound.
This is a beach ruin with a high combat challenge, and good loot.
-Xeno egg
-Handful of minutemen guns
-The pussy wagon
-A tank
-Some puzzle shenanigans
-LIT CONCRETE!!!!!!
-Beach concrete (lit)

## Why It's Good For The Game
We need more beach ruins, and players enjoy large high combat challenge ruins. Plus there's not really another way to get a xeno egg other than one exceedingly rare and not that great lavaland ruin (which should get neutered a bit imo).

## Changelog

:cl:
add: Adds the Xenolab beach ruin
add: Adds /lit and /lit/beach variants for all concrete floors.
/:cl:

### CODE REVIEW

File: _maps/map_catalogue.txt [hunk 1]
```diff
@@ -499,6 +499,10 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 37)(y = 43)(z = 1)
 		Tags = "Medium Combat Challenge", "Medium Loot", "Liveable"
 
+		File Name = "_maps\RandomRuins\BeachRuins\beach_xenolab.dmm"
+		Size = (x = 80)(y = 70)(z = 1)
```
Comment (Erikafox) [inline _maps/map_catalogue.txt]: X should be no greater than 75 otherwise we get _extra_ ruin smushing from ships

**Review by Erikafox — CHANGES_REQUESTED** [review]

File: _maps/map_catalogue.txt (hunk 1 above)
Comment (Latentish) [reply _maps/map_catalogue.txt]: Done.

**Review by Erikafox — CHANGES_REQUESTED** [review]
I'm gonna start at the bottom for this
image
There's absolutely no fluff structurage here, making it incredibly obvious that it's a ruin
image
This does not feel like a believable structure to me. The only way in and out is through the front gate, there's no electrical, there's no atmospherics (not *required* for a beach world, but for a completely sealed building? The mines surrounding it would make it comically hard to access in a safe manner, it's cheesable with 3 bricks of c4 and a pickaxe, or fuck, a pot water grenade. The only decalling in the room is the minutemen star under the tank, there's just a hand grenade and side arm, and a box of stock parts?
Along with double layered blasts which I generally hold to be a mapping sin. If you want it to be harder to get into and stay sealed, set up a proper airlock. Perhaps look into military motorpools for inspiration. 
StrongDMM_VeddMJ7cMP
The dirt suffers the same issue in that it's a monotile, no variation byond the fact that sometimes, there are mines on it. 
StrongDMM_HkrIkdizuB
I have no idea what this is supposed to be. A waiting room? A security checkpoint? The decals on the lower one don't *feel* right to me either
image
utterly decalless room, with the main attraction being weirdly spread apart. Consider making it seem obvious that someone was fucking booking it (or died) when they wrote it. Ditch some of the wood or make it damaged / less thick,
StrongDMM_8cgc6BkZ5X
All of your maints are lightless.
image
you can use a 2x maint loot spawner there, and also spread the stuff out
StrongDMM_2RFgRtJN30
..why is there just a knife?
StrongDMM_TrRFqVuJ8k
This APC is completely unwired. As a matter of fact, the entire base is lacking in wires. Everywhere.
image
Which is strange because you have an engineering area (With a SMES un-terminaled and atop a door). The APC in there will control power for the entire ruin btw.
StrongDMM_QW6kHFHEgE
None of your random junk is pixelshifted.
Every Single Piece Of Ammo you have placed is not spent.
StrongDMM_8qmFdvkqks
why don't you use reinforced walls instead of snowflake concrete. It'd still be tougher to get into, but it lets players use that sandbox element 
StrongDMM_Nm1fGYkp5Y
there are damaged icon states for wooden flooring
image
none of your corpses spawn with clothing
StrongDMM_dqXFOMVVkX
under ALL of your xeno resin (which players will clear out) is monopattern, undecalled floors. They need to have some decals put under them
StrongDMM_8ZMl4Qmhsy
I would rather this be airlocks with blast doors. It'll feel less gamey. 
image
This is the wrong type of 5.56. It's for solgov weapons. See if you can bully apogee into spriting 5.56 and whatever the AK calibre is ammo boxes. 
image
use slab tiles here. 
image
This area needs Way More debris
image
use pavement tiles maybe?
Overall: Needs way more decal work, needs pixel shifting and other fluffing. Should probably be redesigned to mimic a real structure more because it feels less like a ruin and more like a dungeon.

**Comment by Latentish** [comment]
This would basically need to be made again from the ground up.

**Pull request closed without merging**